src/System.Console/src/System/Console.cs

```diff
@@ -25,33 +25,16 @@ public static class Console
         private static ConsoleCancelEventHandler _cancelCallbacks;
         private static ConsolePal.ControlCHandlerRegistrar _registrar;
 
-        internal static T EnsureInitialized<T>(ref T field, Func<T> initializer) where T : class
-        {
-            lock (InternalSyncObject)
-            {
-                T result = Volatile.Read(ref field);
-                if (result == null)
-                {
-                    result = initializer();
-                    Volatile.Write(ref field, result);
-                }
-                return result;
-            }
-        }
+        internal static T EnsureInitializedWrapper<T>(ref T field, Func<T> initializer) where T : class =>
```

Comment:
If we keep the method, I would prefer it just be named `EnsureInitialized`.  Callers of the method don't care how it's implemented and that it happens to be a wrapper for LazyInitialized.EnsureInitialized, all they care is that it's ensuring that the thing they need is initialized.